Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Profile.hs

```diff
@@ -63,3 +99,96 @@ updatePerson personId req = do
       mbEncEmail
       (req.deviceToken)
   pure APISuccess.Success
+
+updateDefaultEmergencyNumbers ::
+  ( EsqDBFlow m r,
+    EncFlow m r,
+    CacheFlow m r,
+    HasFlowEnv m r '["smsCfg" ::: SmsConfig],
+    CoreMetrics m
+  ) =>
+  Id Person.Person ->
+  UpdateProfileDefaultEmergencyNumbersReq ->
+  m UpdateProfileDefaultEmergencyNumbersResp
+updateDefaultEmergencyNumbers personId req = do
+  person <- QPerson.findById personId >>= fromMaybeM (PersonNotFound personId.getId)
```

Comment:
run in replica.

Reply:
> run in replica.

It is POST EP, why it have to be in replica?